report.md

```diff
@@ -10,97 +10,101 @@ description: >-
 
 <h2 id="executive-summary">Executive Summary</h2>
 
-Geospatial information on the Web is fragmented, inadequately covered by Web standards, and as a result lacking interoperability significantly raising barriers to its effective use. Efforts to standardize geoinformation have focused on established geospatial developers, which thus leaves out most people. An opportunity exists today to standardize maps and spatial information in HTML, lowering barriers for everyone, ranging from web developers, GIS data providers, map solution providers, the disabled community to the commercial stakeholders. Browser makers are more open than ever to incorporating ideas and code from community initiatives. It falls to communities to organize themselves to assert the need for and definition of such standards. The global geospatial information community can and should mobilize to simplify and standardize geoinformation for the Web, as maps in HTML, supporting both geographic and non-georeferenced map rendering and interaction.

```

Comment:
Why delete this:

> Browser makers are more open than ever to incorporating ideas and code from community initiatives. It falls to communities to organize themselves to assert the need for and definition of such standards. The global geospatial information community can and should mobilize to simplify and standardize geoinformation for the Web, as maps in HTML, supporting both geographic and non-georeferenced map rendering and interaction.

?  Maybe it went somewhere I haven't noticed yet...

Reply:
It's still there, somewhat tempered, in the next paragraph:

> Browser makers are more open than ever to incorporating ideas and code when the web community establishes clear demand for a particular solution.

Reply:
The diff isn't really very helpful at tracking changes here.